pkg/pkgmgr/rpm.go

```diff
@@ -797,6 +822,87 @@ func (rm *rpmManager) unpackAndMergeUpdates(ctx context.Context, updates unversi
 	return &merged, resultBytes, nil
 }
 
+func (rm *rpmManager) zypperChrootInstallUpdates(ctx context.Context, updates unversioned.UpdatePackages, toolImage string, platform *ocispecs.Platform) (*llb.State, []byte, error) {
+	pkgs := ""
+
+	// Spin up a build tooling container to fetch and install packages
+	// inside a chroot directory to create the patch layer.
+
```

Comment:
thanks for updating the PR with the combined logic! I had a question here - are all SUSE images distroless without the zypper pkgmgr? should we add a check to see if it already exists?

Reply:
got it this makes sense and I think we can keep it this way for simplicity